integrations/Sentry/utils.js

```diff
@@ -0,0 +1,70 @@
+import logger from "../../utils/logUtil";
+
+const convertObjectToArray = (objectInput) => {
+  const list = [];
+  objectInput.forEach(function (objectItem) {
+    const x = objectItem.eventCustomProperties;
+    list.push(x);
+  });
+  return list;
```

Comment:
can we do this in oneliner? 
eg: `return objectInput.map(objectItem => objectItem.eventCustomProperties)`

Reply:
done